src/chainparams.cpp

```diff
@@ -56,7 +56,7 @@ class CMainParams : public CChainParams {
 
         // Timestamps for forking consensus rule changes:
         // Allow bigger blocks if:
-        consensus.nActivateSizeForkMajority = 750; // 75% of hashpower to activate fork
+        consensus.nActivateSizeForkMajority = 800; // 80% of hashpower to activate fork
```

Comment:
The comment is actually not accurate. Even with just 77% hashpower there's a 60% chance of activating within 90 days.
I think it's clear enough if the comment says  // of 1000 most recent blocks